testing/matrix-sdk-integration-testing/src/tests/sliding_sync/room.rs

```diff
@@ -538,3 +550,299 @@ async fn test_room_notification_count() -> Result<()> {
 
     Ok(())
 }
+
+static DROP_TODEVICE: StdMutex<bool> = StdMutex::new(true);
+fn drop_todevice(response: &mut Bytes) {
```

Comment:
Please comment what this does. Even if this is test-only code, we need to understand in 6 months what it does at a high-level view.